packages/editor/src/client/panel/embedEditPanel.tsx

```diff
@@ -114,29 +117,33 @@ export function EmbedEditPanel({value, onClose, onConfirm}: EmbedEditPanel) {
       <PanelSection>
         <TextArea
           marginBottom={Spacing.ExtraSmall}
-          label="Embed"
+          label={t('Embed')}
           errorMessage={errorMessage}
           value={input}
           onChange={e => setInput(e.target.value)}
         />
         <Box marginBottom={Spacing.ExtraSmall}>
           <Typography variant="subtitle1" spacing="small">
-            Facebook Post/Video, Instagram Post, Twitter Tweet, Vimeo Video, YouTube Video formatted
-            like e.g.
+            {t(
+              'Facebook Post/Video, Instagram Post, Twitter Tweet, Vimeo Video, YouTube Video formatted like e.g.'
+            )}
           </Typography>
           <code>https://www.facebook.com/id/posts/id/</code>
           <Typography variant="subtitle1" spacing="small">
-            Embed codes attributes <code>title, src, width, height</code> are validated e.g.
+            {t('Embed codes attributes <code>title, src, width, height</code> are validated e.g.')}
           </Typography>
           <code>{'<iframe width="560" height="315" src="https://..."></iframe>'}</code>
           <Typography variant="subtitle1" spacing="small">
-            Set <code>style</code> alternatively as attribute to overwrite the defaults and disable
-            auto ratio size scaling .e.g.
+            {t(
+              'Set <code>style</code> alternatively as attribute to overwrite the defaults and disable auto ratio size scaling .e.g.'
+            )}
           </Typography>
+
           <code>{'<iframe style="height:350px;width:100%;" src="https://..."></iframe>'}</code>
```

Comment:
Why didn't you put this string into the translations?

Reply:
Because it's pure code and there's nothing to translate. I thought it wouldn't make sense to populate the dictionary with this

Reply:
If it's text that is displayed to the users it should be in the translation files. The name translations is a bit confusing here.

Reply:
Done